Add strategy templates (#21)
croct-tech/templates: templates/croct/experience/e-commerce/instagram-campaign/README.md

# Introduction

> This template creates an experience to personalize your e-commerce website for visitors from Instagram paid campaigns interested in specific services or offerings.

Reviewer: ```suggestion
This template creates an experience to personalize an e-commerce website for visitors coming from Instagram paid campaigns.
```
